src/vrapp/VRVolumeApp.cpp

```diff
@@ -40,28 +42,43 @@
 #include <sstream> 
 #include "GLMLoader.h"
 #include <locale>
-
+#include <algorithm>
 
 #include <glm/gtx/euler_angles.hpp>
 
+#include "Model.h"
+#include "Texture.h"
+
 
 VRVolumeApp::VRVolumeApp() :m_mesh_model(nullptr), m_clip_max{ 1.0f }, m_clip_min{ 0.0f }, m_clip_ypr{ 0.0f }, m_clip_pos{ 0.0 }, m_wasd_pressed(0),
 m_lookingGlass(false), m_isInitailized(false), m_speed(0.01f), m_movieAction(nullptr), m_moviename("movie.mp4"), m_noColor(0.0f),
 m_ambient(0.2f, 0.2f, 0.2f, 1.0f), m_diffuse(0.5f, 0.5f, 0.5f, 1.0f), m_ui_view(nullptr), m_animated(false), m_numVolumes(0), m_selectedVolume(0),
 m_multiplier(1.0f), m_threshold(0.0f), m_frame(0.0f), m_use_multi_transfer(false), m_clipping(false), m_show_menu(true),
-m_window_properties(nullptr)
+m_window_properties(nullptr), m_animation_speed(1.0f), m_current_movie_state(MOVIE_STOP), m_app_mode(MANUAL), m_end_load(false)
 {
   m_renders.push_back(new VolumeSliceRenderer());
   m_renders.push_back(new VolumeRaycastRenderer());
 }
 
+
+
 VRVolumeApp::~VRVolumeApp()
 {
   if (m_ui_view)
   {
     delete m_ui_view;
   }
```

Comment:
```suggestion
  if (m_ui_view) delete m_ui_view;
```

Reply:
the formatter replaces your line as:
```
if (m_ui_view)
    delete m_ui_view;
```

Reply:
Oh okay that's perfectly fine too! Things are looking pretty pristine now that the formatter is running 👌🏼